usr/bin/nxdk-link

```diff
@@ -12,4 +12,11 @@ fi
 
 ${LLD_LINK} -subsystem:windows -fixed:no -entry:XboxCRTEntry \
 -stack:${NXDK_STACKSIZE} -safeseh:no \
+${NXDK_DIR}/lib/libwinapi.lib \
+${NXDK_DIR}/lib/xboxkrnl/libxboxkrnl.lib \
+${NXDK_DIR}/lib/libxboxrt.lib \
+${NXDK_DIR}/lib/libpdclib.lib \
+${NXDK_DIR}/lib/libnxdk_hal.lib \
+${NXDK_DIR}/lib/libnxdk.lib \
+${NXDK_DIR}/lib/libnxdk_usb.lib \
```

Comment:
These should be disabled if the flags claim that this should be freestanding (or something).

This is also why I'd prefer to write these wrappers in Python: Deal with more complicated argument logic.